src/libs/antares/study/area/area.cpp

```diff
@@ -46,23 +46,22 @@ void Area::internalInitialize()
 }
 
 Area::Area() :
- index((uint)(-1)),
- enabled(true),
- reserves(fhrMax, HOURS_PER_YEAR),
- miscGen(fhhMax, HOURS_PER_YEAR),
- nodalOptimization(anoAll),
- spreadUnsuppliedEnergyCost(0.),
- spreadSpilledEnergyCost(0.),
- filterSynthesis(filterAll),
- filterYearByYear(filterAll),
- ui(nullptr),
- nbYearsInParallel(0),
- invalidateJIT(false)
+     index((uint)(-1)),
+     enabled(true),
+     reserves(fhrMax, HOURS_PER_YEAR),
+     miscGen(fhhMax, HOURS_PER_YEAR),
+     nodalOptimization(anoAll),
+     spreadUnsuppliedEnergyCost(0.),
+     spreadSpilledEnergyCost(0.),
+     filterSynthesis(filterAll),
+     filterYearByYear(filterAll),
+     ui(nullptr),
+     invalidateJIT(false)
```

Comment:
Set constants in area.h to avoid code duplication / mistakes. Leave `reserves` & `miscGen` here.